src/bundle/Resources/public/scss/ezplatform.scss

```diff
@@ -1,5 +1,6 @@
 @import 'custom';
 @import 'general';
+@import 'mixins';
```

Comment:
I think we should load `mixins` before `general`.

Reply:
I don't see a point in moving it right now. In the future we can do it if required.